test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -999,10 +999,19 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 	})
 })
 
-func installNvidiaDriversDaemonSet(namespace string) {
+func installNVIDIADriversDaemonSet(f *framework.Framework) {
 	ginkgo.By("Add daemonset which installs nvidia drivers")
-	// the link differs from one in GKE documentation; discussed with @mindprince this one should be used
-	framework.RunKubectlOrDie(namespace, "apply", "-f", "https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/daemonset.yaml")
+
+	dsYamlUrl := "https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/daemonset.yaml"
+	framework.Logf("Using %v", dsYamlUrl)
+	// Creates the DaemonSet that installs Nvidia Drivers.
+	ds, err := framework.DsFromManifest(dsYamlUrl)
+	framework.ExpectNoError(err)
+	ds.Namespace = f.Namespace.Name
+
+	_, err = f.ClientSet.AppsV1().DaemonSets(f.Namespace.Name).Create(context.TODO(), ds, metav1.CreateOptions{})
+	framework.ExpectNoError(err, "failed to create nvidia-driver-installer daemonset")
+	framework.Logf("Successfully created daemonset to install Nvidia drivers.")
```

Comment:
this success line (Logf) can be removed.

Reply:
sure